pom.xml

```diff
@@ -414,7 +414,11 @@
                 <artifactId>wildfly-elytron-http-sso</artifactId>
                 <version>${project.version}</version>
             </dependency>
-
+            <dependency>
```

Comment:
Just a minor comment, the dependencies in each pom.xml file should be in alphabetical order

Reply:
Fixed